syli9526/ArchitectureStudy/app/src/main/java/com/example/architecturestudy/network/ApiService.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.architecturestudy.network
+
+import com.example.architecturestudy.model.MarketResponse
+import com.example.architecturestudy.model.TickerResponse
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface ApiService {
+    @GET("/v1/market/all")
+    fun getMarketData(): Call<List<MarketResponse>>
+
+    @GET("/v1/ticker?")
```

Comment:
Query 의 경우 retrofit 에서 자체적으로 ? 를 붙여주도록 되어있기 때문에 생략 가능합니다
```suggestion
    @GET("/v1/ticker")
```

Reply:
[975ad5d](https://github.com/StudyFork/GoogryAndroidArchitectureStudy/pull/184/commits/975ad5d3671c73f54924ca529f8afb478daee7ee)
수정하였습니다.